fastplotlib/graphics/line.py

```diff
@@ -53,3 +56,32 @@ def update_colors(self, colors: np.ndarray):
 
         self.world_object.geometry.colors.data[:] = self.colors
         self.world_object.geometry.colors.update_range()
+
+    @property
+    def indices(self) -> Any:
+        return self.indices
+
+    @property
+    def features(self) -> List[str]:
+        return self.features
+
+    def _set_feature(self, name: str, new_data: Any, indices: Any):
+        if name == "color":
```

Comment:
Use getattr instead of if statements

Reply:
I don't understand this one

Reply:
https://docs.python.org/3/library/functions.html#getattr

```python3
update_func = gettattr(self, f"update_{name}")
update_func(new_data, indices=indices)
```

Reply:
you will need to implement a way for the update functions to use the indices